src/features/cheatcodes/pages/AppComponents.tsx

```diff
@@ -79,7 +196,10 @@ export const AppComponents: FunctionComponent = () => {
         <Close size={24} />
         <Text>{_(t` - Close `)}</Text>
       </AlignedText>
-      <Spacer.Column numberOfSpaces={1} />
+      <AlignedText>
+        <PassCulture size={24} />
+        <Text>{_(t` - PassCulture `)}</Text>
```

Comment:
```suggestion
        <Text>{' - PassCulture ')}</Text>
```
Je pense qu'il faudrait retirer lingui de cette page. Cela cause plus de bruits qu'autre chose. 
A priori, on peut tout écrire en français ici, sans utiliser lingui.